AndroidAnnotations/functional-test-1-5/src/main/java/org/androidannotations/test15/ActivityWithServices.java

```diff
@@ -121,6 +125,9 @@
 	@SystemService
 	DropBoxManager dropBoxManager;
 	
+	// @SystemService
+	// EthernetManager ethernetManager; // hidden API
```

Comment:
Actually i just noticed that i missed some of the [new hidden services](https://github.com/android/platform_frameworks_base/blob/master/core/java/android/content/Context.java). I will add them later today.